src/Protocol/DFRN.php

```diff
@@ -2628,6 +2633,11 @@ public static function import($xml, $importer, $sort_by_date = false, $protocol
 		$header["wall"] = 0;
 		$header["origin"] = 0;
 		$header["contact-id"] = $importer["id"];
+		$header["direction"] = $direction;
+
+		if ($direction == Conversation::RELAY) {
```

Comment:
Please use `===` when checking against a constant value.

Reply:
Why? Is it in some way faster?

Reply:
BTW: done

Reply:
Since you're comparing constant values with constant values, you can be sure types won't be different. It's marginally faster but it's most importantly safer.